dev/190f7529-eb16-4963-9e23-78b56f45c9e1/device.lp

```diff
@@ -0,0 +1,59 @@
+(librepcb_device 190f7529-eb16-4963-9e23-78b56f45c9e1
+ (name "RN2483")
+ (description "Low-Power Long Range LoRa Technology Transceiver Module.")
+ (keywords "rn2483,rn2483a,lorawan,lora,transceiver")
+ (author "Danilo B.")
+ (version "0.1")
+ (created 2019-01-27T00:48:03Z)
+ (deprecated false)
+ (category f7f9bc34-7966-4af4-8cd4-4b8101e0823a)
```

Comment:
Shouldn't this have the `4e89bf79-0506-4b24-8616-b0461a5932e0` which is created below?